mora/service/employee.py

```diff
@@ -340,7 +340,7 @@ def create_employee_relation(employee_uuid):
     :<jsonarr string type: **"manager"**
     :<jsonarr string org_unit: The associated org unit
     :<jsonarr string manager_type: The manager type
-    :<jsonarr string responsibility: The manager responsibility
+    :<jsonarr array responsibilities: The manager responsibilities
```

Comment:
Isn't this still called `responsibility`?

Reply:
Yup.